migration-from-4x.md

```diff
@@ -298,8 +300,10 @@ For example:
 
 ```js
 editor.addButton('mybutton', {
-  text: "My Button",
-  onclick: () => alert("My Button clicked!")
+  text: 'My Button',
+  onclick: function () {
+    alert('My Button clicked!')
```

Comment:
Missing semicolon at the end of the line here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d